[100%] Testing the TCP connection...
Test project /mnt/i/DIY/CS144/sponge/build
        Start   1: t_wrapping_ints_cmp
  1/159 Test   #1: t_wrapping_ints_cmp ..............   Passed    0.01 sec
        Start   2: t_wrapping_ints_unwrap
  2/159 Test   #2: t_wrapping_ints_unwrap ...........   Passed    0.01 sec
        Start   3: t_wrapping_ints_wrap
  3/159 Test   #3: t_wrapping_ints_wrap .............   Passed    0.02 sec
        Start   4: t_recv_connect
  4/159 Test   #4: t_recv_connect ...................   Passed    0.03 sec
        Start   5: t_recv_transmit
  5/159 Test   #5: t_recv_transmit ..................   Passed    0.29 sec
        Start   6: t_recv_window
  6/159 Test   #6: t_recv_window ....................   Passed    0.03 sec
        Start   7: t_recv_reorder
  7/159 Test   #7: t_recv_reorder ...................   Passed    0.04 sec
        Start   8: t_recv_close
  8/159 Test   #8: t_recv_close .....................   Passed    0.03 sec
        Start   9: t_send_connect
  9/159 Test   #9: t_send_connect ...................   Passed    0.03 sec
        Start  10: t_send_transmit
 10/159 Test  #10: t_send_transmit ..................   Passed    0.31 sec
        Start  11: t_send_retx
 11/159 Test  #11: t_send_retx ......................   Passed    0.04 sec
        Start  12: t_send_window
 12/159 Test  #12: t_send_window ....................   Passed    0.19 sec
        Start  13: t_send_ack
 13/159 Test  #13: t_send_ack .......................   Passed    0.03 sec
        Start  14: t_send_close
 14/159 Test  #14: t_send_close .....................   Passed    0.03 sec
        Start  15: t_strm_reassem_single
 15/159 Test  #15: t_strm_reassem_single ............   Passed    0.02 sec
        Start  16: t_strm_reassem_seq
 16/159 Test  #16: t_strm_reassem_seq ...............   Passed    0.02 sec
        Start  17: t_strm_reassem_dup
 17/159 Test  #17: t_strm_reassem_dup ...............   Passed    0.05 sec
        Start  18: t_strm_reassem_holes
 18/159 Test  #18: t_strm_reassem_holes .............   Passed    0.02 sec
        Start  19: t_strm_reassem_many
 19/159 Test  #19: t_strm_reassem_many ..............   Passed    7.52 sec
        Start  20: t_strm_reassem_overlapping
 20/159 Test  #20: t_strm_reassem_overlapping .......   Passed    0.03 sec
        Start  21: t_strm_reassem_win
 21/159 Test  #21: t_strm_reassem_win ...............   Passed    9.31 sec
        Start  22: t_strm_reassem_cap
 22/159 Test  #22: t_strm_reassem_cap ...............   Passed    0.58 sec
        Start  23: t_byte_stream_construction
 23/159 Test  #23: t_byte_stream_construction .......   Passed    0.02 sec
        Start  24: t_byte_stream_one_write
 24/159 Test  #24: t_byte_stream_one_write ..........   Passed    0.03 sec
        Start  25: t_byte_stream_two_writes
 25/159 Test  #25: t_byte_stream_two_writes .........   Passed    0.03 sec
        Start  26: t_byte_stream_capacity
 26/159 Test  #26: t_byte_stream_capacity ...........   Passed    5.16 sec
        Start  27: t_byte_stream_many_writes
 27/159 Test  #27: t_byte_stream_many_writes ........   Passed    0.05 sec
        Start  28: t_webget
 28/159 Test  #28: t_webget .........................   Passed    1.80 sec
        Start  31: t_tcp_parser
 29/159 Test  #31: t_tcp_parser .....................   Passed    0.10 sec
        Start  32: t_ipv4_parser
 30/159 Test  #32: t_ipv4_parser ....................   Passed    0.07 sec
        Start  33: t_active_close
 31/159 Test  #33: t_active_close ...................***Failed    0.06 sec
Test Failure on expectation:
	Expectation: TCP in state sender=`stream finished and fully acknowledged`, receiver=`input to stream has ended`, active=1, linger_after_streams_finish=1

Failure message:
	The TCP was in state `sender=`stream ongoing`, receiver=`input to stream has ended`, active=1, linger_after_streams_finish=1`, but it was expected to be in state `sender=`stream finished and fully acknowledged`, receiver=`input to stream has ended`, active=1, linger_after_streams_finish=1`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      close
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=1,ackno=1,seqno=1,)
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=2,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)
	Action:      9999ms pass

Warning: Unclean shutdown of TCPConnection
The TCP was in state `sender=`stream ongoing`, receiver=`input to stream has ended`, active=1, linger_after_streams_finish=1`, but it was expected to be in state `sender=`stream finished and fully acknowledged`, receiver=`input to stream has ended`, active=1, linger_after_streams_finish=1`

        Start  34: t_passive_close
 32/159 Test  #34: t_passive_close ..................***Failed    0.07 sec
Test Failure on expectation:
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2,)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=0,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=0,ack=1,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=1,payload_size=0,)
	Action:      packet arrives: Header(flags=AF,seqno=1,ack=1,win=137) with no payload

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  36: t_ack_rst
 33/159 Test  #36: t_ack_rst ........................***Failed    0.08 sec
Test 1
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  38: t_ack_rst_win
 34/159 Test  #38: t_ack_rst_win ....................***Failed    0.07 sec
Test Failure on expectation:
	Expectation: TCP has 0 unassembled bytes

Failure message:
	The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2147483647,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2147483647,ack=2147483648,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2147483648,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483648,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483648,ack=2147483647,win=137) with no payload
	Expectation: no (more) segments sent
	Action:      packet arrives: Header(flags=A,seqno=2147483647,ack=2147483648,win=137) with 1 data bytes "<1>"

Note:
	test 1 failed: seg queued on early seqno

Warning: Unclean shutdown of TCPConnection
The TCP has `unassembled_bytes = 1`, but unassembled_bytes was expected to be `0`

        Start  40: t_connect
 35/159 Test  #40: t_connect ........................   Passed    0.07 sec
        Start  42: t_listen
 36/159 Test  #42: t_listen .........................   Passed    0.04 sec
        Start  43: t_winsize
 37/159 Test  #43: t_winsize ........................***Failed    0.05 sec
Test Failure on expectation:
	Action:      write (25026 bytes with 25026 accepted) [3d, 8d, f0, 62, 96, 28, fe, 57, 10, e4, 94, 1f, 57, c0, b4, f4, ...]

Failure message:
	25026 bytes should have been written but 0 were

List of steps that executed successfully:
	Action:      listen
	Action:      packet arrives: Header(flags=S,seqno=3690884089,ack=0,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,ackno=3690884090,win=29050,)
	Action:      packet arrives: Header(flags=A,seqno=3690884090,ack=3518522929,win=8342) with no payload
	Expectation: no (more) segments sent
	Expectation: TCP in state sender=`stream ongoing`, receiver=`SYN received (ackno exists), and input to stream hasn't ended`, active=1, linger_after_streams_finish=1

Warning: Unclean shutdown of TCPConnection
25026 bytes should have been written but 0 were

        Start  45: t_retx
 38/159 Test  #45: t_retx ...........................***Failed    0.12 sec
  check_segment
Test Failure on expectation:
	Expectation: segment sent with (A=1,payload_size=4,data="asdf",)

Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=4092246550,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=4092246550,ack=4092246551,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=4092246551,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not (in check_segment called from line 36)

        Start  46: t_retx_win
 39/159 Test  #46: t_retx_win .......................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=3981983884,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=3981983884,ack=3981983885,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=3981983885,payload_size=0,)
	Action:      write (4 bytes) [61, 73, 64, 66]
	Action:      1ms pass
	Action:      20ms pass
	Action:      write (4 bytes) [71, 77, 65, 72]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()s

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  47: t_loopback
 40/159 Test  #47: t_loopback .......................***Failed    0.06 sec
Test Failure on expectation:
	Expectation: TCP has 4080 bytes in flight

Failure message:
	The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `4080`

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=2777361097,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=2777361097,ack=2777361098,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=2777361098,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=2777361098,ack=2777361098,win=65000) with no payload
	Action:      write (4080 bytes) [6f, b8, bf, 38, 4c, 43, 72, da, 40, 41, 82, 89, d7, 29, 2f, 2f, ...]
	Action:      1ms pass

Warning: Unclean shutdown of TCPConnection
The TCP has `bytes_in_flight = 0`, but bytes_in_flight was expected to be `4080`

        Start  48: t_loopback_win
 41/159 Test  #48: t_loopback_win ...................***Failed    0.05 sec
Test Failure on expectation:
	Expectation: segment sent

Failure message:
	The TCP should have produces a segment, but did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=3410526289,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=3410526289,ack=3410526290,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=3410526290,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=3410526290,ack=3410526290,win=65000) with no payload
	Action:      write (703 bytes) [ea, a2, bb, e6, 15, 16, dc, 27, d5, 59, 16, 3a, 1b, ac, d3, 37, ...]
	Action:      1ms pass

Note:
	test 2 failed: cannot read after write()

Warning: Unclean shutdown of TCPConnection
The TCP should have produces a segment, but did not

        Start  49: t_reorder
 42/159 Test  #49: t_reorder ........................***Failed    0.05 sec
Test Failure on expectation:
	segment sent with (A=1,)
Failure message:
	The TCP should have produced a segment that existed, but it did not

List of steps that executed successfully:
	Action:      connect
	Expectation: exactly one segment sent with (A=0,R=0,S=1,F=0,seqno=1981632363,payload_size=0,)
	Action:      packet arrives: Header(flags=SA,seqno=960682419,ack=1981632364,win=137) with no payload
	Expectation: exactly one segment sent with (A=1,R=0,S=0,F=0,ackno=960682420,payload_size=0,)
	Action:      packet arrives: Header(flags=A,seqno=960709981,ack=1981632364,win=137) with 865 data bytes "8<132><218>BZ<218><148><229><150><132><127><153><222><150>7<179>..."

Note:
	test 1 failed: no ACK for datagram

Warning: Unclean shutdown of TCPConnection
The TCP should have produced a segment that existed, but it did not

        Start  50: t_address_dt
 43/159 Test  #50: t_address_dt .....................   Passed    0.04 sec
        Start  51: t_parser_dt
 44/159 Test  #51: t_parser_dt ......................   Passed    0.02 sec
        Start  52: t_socket_dt
 45/159 Test  #52: t_socket_dt ......................   Passed    0.03 sec
        Start  53: t_udp_client_send
 46/159 Test  #53: t_udp_client_send ................***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:6106... done.
new connection from 169.254.144.1:57951.
DEBUG: Outbound stream to 169.254.144.1:57951 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:6106 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1509 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  54: t_udp_server_send
 47/159 Test  #54: t_udp_server_send ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5435... done.
new connection from 169.254.144.1:44419.
DEBUG: Outbound stream to 169.254.144.1:5435 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:44419 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  55: t_udp_client_recv
 48/159 Test  #55: t_udp_client_recv ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:12965... done.
new connection from 169.254.144.1:48168.
DEBUG: Outbound stream to 169.254.144.1:12965 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:48168 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  56: t_udp_server_recv
 49/159 Test  #56: t_udp_server_recv ................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31182... done.
new connection from 169.254.144.1:55406.
DEBUG: Outbound stream to 169.254.144.1:55406 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1598 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  57: t_udp_client_dupl
 50/159 Test  #57: t_udp_client_dupl ................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25021... done.
new connection from 169.254.144.1:55102.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised lengthException in TCPConnection runner thread: 
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1617 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  58: t_udp_server_dupl
 51/159 Test  #58: t_udp_server_dupl ................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:4828... done.
new connection from 169.254.144.1:59304.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1639 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  59: t_ucS_1M_32k
 52/159 Test  #59: t_ucS_1M_32k .....................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22986... done.
new connection from 169.254.144.1:49951.
DEBUG: Outbound stream to 169.254.144.1:49951 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:22986 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1659 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  60: t_ucS_128K_8K
 53/159 Test  #60: t_ucS_128K_8K ....................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11045... done.
new connection from 169.254.144.1:36192.
DEBUG: Outbound stream to 169.254.144.1:36192 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:11045 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1679 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  61: t_ucS_16_1
 54/159 Test  #61: t_ucS_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:4000... done.
new connection from 169.254.144.1:50703.
DEBUG: Outbound stream to 169.254.144.1:50703 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1699 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  62: t_ucS_32K_d
 55/159 Test  #62: t_ucS_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:25214... done.
new connection from 169.254.144.1:46081.
DEBUG: Outbound stream to 169.254.144.1:46081 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:25214 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1719 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  63: t_ucR_1M_32k
 56/159 Test  #63: t_ucR_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31718... done.
new connection from 169.254.144.1:52057.
DEBUG: Outbound stream to 169.254.144.1:31718 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:52057 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  64: t_ucR_128K_8K
 57/159 Test  #64: t_ucR_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:8477... done.
new connection from 169.254.144.1:34151.
DEBUG: Outbound stream to 169.254.144.1:8477 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:34151 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  65: t_ucR_16_1
 58/159 Test  #65: t_ucR_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11119... done.
new connection from 169.254.144.1:45133.
DEBUG: Outbound stream to 169.254.144.1:11119 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  66: t_ucR_32K_d
 59/159 Test  #66: t_ucR_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:28981... done.
new connection from 169.254.144.1:33729.
DEBUG: Outbound stream to 169.254.144.1:28981 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:33729 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  67: t_ucD_1M_32k
 60/159 Test  #67: t_ucD_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:7759... done.
new connection from 169.254.144.1:47041.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised lengthException in TCPConnection runner thread: 
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1869 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1871 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  68: t_ucD_128K_8K
 61/159 Test  #68: t_ucD_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32092... done.
new connection from 169.254.144.1:56357.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1890 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1892 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  69: t_ucD_16_1
 62/159 Test  #69: t_ucD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:9098... done.
new connection from 169.254.144.1:53388.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1911 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1913 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  70: t_ucD_32K_d
 63/159 Test  #70: t_ucD_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:3912... done.
new connection from 169.254.144.1:50077.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1930 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  1932 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  71: t_usS_1M_32k
 64/159 Test  #71: t_usS_1M_32k .....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:26904... done.
new connection from 169.254.144.1:38479.
DEBUG: Outbound stream to 169.254.144.1:26904 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:38479 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  72: t_usS_128K_8K
 65/159 Test  #72: t_usS_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:19536... done.
new connection from 169.254.144.1:43493.
DEBUG: Outbound stream to 169.254.144.1:19536 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:43493 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  73: t_usS_16_1
 66/159 Test  #73: t_usS_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32071... done.
new connection from 169.254.144.1:41824.
DEBUG: Outbound stream to 169.254.144.1:32071 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:41824 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  74: t_usS_32K_d
 67/159 Test  #74: t_usS_32K_d ......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:9761... done.
new connection from 169.254.144.1:54657.
DEBUG: Outbound stream to 169.254.144.1:9761 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:54657 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  75: t_usR_1M_32k
 68/159 Test  #75: t_usR_1M_32k .....................***Failed    0.25 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:19048... done.
new connection from 169.254.144.1:60915.
DEBUG: Outbound stream to 169.254.144.1:60915 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:19048 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2083 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  76: t_usR_128K_8K
 69/159 Test  #76: t_usR_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:20454... done.
new connection from 169.254.144.1:51334.
DEBUG: Outbound stream to 169.254.144.1:51334 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:20454 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2103 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  77: t_usR_16_1
 70/159 Test  #77: t_usR_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23292... done.
new connection from 169.254.144.1:49145.
DEBUG: Outbound stream to 169.254.144.1:49145 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2123 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  78: t_usR_32K_d
 71/159 Test  #78: t_usR_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:28121... done.
new connection from 169.254.144.1:39025.
DEBUG: Outbound stream to 169.254.144.1:39025 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:28121 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2141 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  79: t_usD_1M_32k
 72/159 Test  #79: t_usD_1M_32k .....................***Failed    0.26 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:22588... done.
new connection from 169.254.144.1:35174.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2162 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156:  2160 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start  80: t_usD_128K_8K
 73/159 Test  #80: t_usD_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11024... done.
new connection from 169.254.144.1:47662.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2186 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  81: t_usD_16_1
 74/159 Test  #81: t_usD_16_1 .......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32527... done.
new connection from 169.254.144.1:53091.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2205 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2207 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  82: t_usD_32K_d
 75/159 Test  #82: t_usD_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23027... done.
new connection from 169.254.144.1:36915.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2224 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2226 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  83: t_ucS_128K_8K_l
 76/159 Test  #83: t_ucS_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:4901... done.
new connection from 169.254.144.1:54287.
DEBUG: Outbound stream to 169.254.144.1:54287 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:4901 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2246 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  84: t_ucS_128K_8K_L
 77/159 Test  #84: t_ucS_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:5157... done.
new connection from 169.254.144.1:60758.
DEBUG: Outbound stream to 169.254.144.1:60758 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:5157 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2266 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  85: t_ucS_128K_8K_lL
 78/159 Test  #85: t_ucS_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:23748... done.
new connection from 169.254.144.1:49082.
DEBUG: Outbound stream to 169.254.144.1:49082 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:23748 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2286 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  86: t_ucR_128K_8K_l
 79/159 Test  #86: t_ucR_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31991... done.
new connection from 169.254.144.1:37923.
DEBUG: Outbound stream to 169.254.144.1:31991 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:37923 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  87: t_ucR_128K_8K_L
 80/159 Test  #87: t_ucR_128K_8K_L ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:14128... done.
new connection from 169.254.144.1:58540.
DEBUG: Outbound stream to 169.254.144.1:14128 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:58540 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  88: t_ucR_128K_8K_lL
 81/159 Test  #88: t_ucR_128K_8K_lL .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29542... done.
new connection from 169.254.144.1:48418.
DEBUG: Outbound stream to 169.254.144.1:29542 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:48418 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  89: t_ucD_128K_8K_l
 82/159 Test  #89: t_ucD_128K_8K_l ..................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:32438... done.
new connection from 169.254.144.1:34745.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2404 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2406 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  90: t_ucD_128K_8K_L
 83/159 Test  #90: t_ucD_128K_8K_L ..................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31972... done.
new connection from 169.254.144.1:57935.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2430 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  91: t_ucD_128K_8K_lL
 84/159 Test  #91: t_ucD_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:16261... done.
new connection from 169.254.144.1:35043.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2449 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2451 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  92: t_usS_128K_8K_l
 85/159 Test  #92: t_usS_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:7285... 

        Start  93: t_usS_128K_8K_L
 86/159 Test  #93: t_usS_128K_8K_L ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:18592... done.
DEBUG: Outbound stream to 169.254.144.1:18592 finished (0 bytes still in flight).
new connection from 169.254.144.1:54176.
DEBUG: Inbound stream from 169.254.144.1:54176 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  94: t_usS_128K_8K_lL
 87/159 Test  #94: t_usS_128K_8K_lL .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:29651... done.
new connection from 169.254.144.1:46506.
DEBUG: Outbound stream to 169.254.144.1:29651 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:46506 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start  95: t_usR_128K_8K_l
 88/159 Test  #95: t_usR_128K_8K_l ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:19232... done.
new connection from 169.254.144.1:36992.
DEBUG: Outbound stream to 169.254.144.1:36992 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:19232 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2580 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  96: t_usR_128K_8K_L
 89/159 Test  #96: t_usR_128K_8K_L ..................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:11096... done.
new connection from 169.254.144.1:45338.
DEBUG: Outbound stream to 169.254.144.1:45338 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:11096 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2600 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  97: t_usR_128K_8K_lL
 90/159 Test  #97: t_usR_128K_8K_lL .................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:15075... done.
new connection from 169.254.144.1:43609.
DEBUG: Outbound stream to 169.254.144.1:43609 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.1:15075 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2620 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  98: t_usD_128K_8K_l
 91/159 Test  #98: t_usD_128K_8K_l ..................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:31778... done.
new connection from 169.254.144.1:56078.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2641 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start  99: t_usD_128K_8K_L
 92/159 Test  #99: t_usD_128K_8K_L ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:3729... done.
new connection from 169.254.144.1:35245.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2662 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 100: t_usD_128K_8K_lL
 93/159 Test #100: t_usD_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.1:19484... done.
new connection from 169.254.144.1:48574.
Exception in TCPConnection runner thread: Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2681 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2683 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 101: t_ipv4_client_send
 94/159 Test #101: t_ipv4_client_send ...............***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:15784... done.
new connection from 169.254.144.9:35195.
DEBUG: Outbound stream to 169.254.144.9:35195 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2701 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 102: t_ipv4_server_send
 95/159 Test #102: t_ipv4_server_send ...............***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:21161... done.
DEBUG: Outbound stream to 169.254.144.9:21161 finished (0 bytes still in flight).
new connection from 169.254.145.9:4655.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 103: t_ipv4_client_recv
 96/159 Test #103: t_ipv4_client_recv ...............***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:21340... done.
DEBUG: Outbound stream to new connection from 169.254.144.9:34369.
169.254.145.9:21340 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 104: t_ipv4_server_recv
 97/159 Test #104: t_ipv4_server_recv ...............***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:16692... done.
new connection from 169.254.145.9:7800.
DEBUG: Outbound stream to 169.254.145.9:7800 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2783 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 105: t_ipv4_client_dupl
 98/159 Test #105: t_ipv4_client_dupl ...............***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:19130... done.
new connection from 169.254.144.9:34170.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2802 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 106: t_ipv4_server_dupl
 99/159 Test #106: t_ipv4_server_dupl ...............***Failed    0.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:27514... done.
new connection from 169.254.145.9:36303.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2821 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 107: t_icS_1M_32k
100/159 Test #107: t_icS_1M_32k .....................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:4418... done.
new connection from 169.254.144.9:20012.
DEBUG: Outbound stream to 169.254.144.9:20012 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:4418 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2841 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 108: t_icS_128K_8K
101/159 Test #108: t_icS_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:28603... done.
new connection from 169.254.144.9:14546.
DEBUG: Outbound stream to 169.254.144.9:14546 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:28603 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2861 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 109: t_icS_16_1
102/159 Test #109: t_icS_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:28415... done.
new connection from 169.254.144.9:35646.
DEBUG: Outbound stream to 169.254.144.9:35646 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2881 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 110: t_icS_32K_d
103/159 Test #110: t_icS_32K_d ......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:9331... done.
new connection from 169.254.144.9:57995.
DEBUG: Outbound stream to 169.254.144.9:57995 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:9331 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  2902 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 111: t_icR_1M_32k
104/159 Test #111: t_icR_1M_32k .....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:30145... done.
DEBUG: Outbound stream to new connection from 169.254.145.9:30145 finished (169.254.144.9:5939.
0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:5939 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 112: t_icR_128K_8K
105/159 Test #112: t_icR_128K_8K ....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:22363... done.
DEBUG: Outbound stream to new connection from 169.254.144.9:54365.
169.254.145.9:22363 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:54365 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 113: t_icR_16_1
106/159 Test #113: t_icR_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:20737... done.
DEBUG: Outbound stream to 169.254.145.9:20737 finished (0 bytes still in flight).
new connection from 169.254.144.9:49544.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 114: t_icR_32K_d
107/159 Test #114: t_icR_32K_d ......................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:33017... done.
DEBUG: Outbound stream to 169.254.145.9:33017 finished (0 bytes still in flight).
new connection from 169.254.144.9:11023.
DEBUG: Inbound stream from 169.254.144.9:11023 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 115: t_icD_1M_32k
108/159 Test #115: t_icD_1M_32k .....................***Failed    0.31 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:25884... done.
new connection from 169.254.144.9:63264.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3051 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 116: t_icD_128K_8K
109/159 Test #116: t_icD_128K_8K ....................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:6628... done.
new connection from 169.254.144.9:34674.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3072 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 117: t_icD_16_1
110/159 Test #117: t_icD_16_1 .......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:8942... done.
new connection from 169.254.144.9:39773.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3093 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 118: t_icD_32K_d
111/159 Test #118: t_icD_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:33540... done.
new connection from 169.254.144.9:53085.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3112 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 119: t_isS_1M_32k
112/159 Test #119: t_isS_1M_32k .....................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:14309... done.
DEBUG: Outbound stream to 169.254.144.9:14309 finished (0 bytes still in flight).
new connection from 169.254.145.9:58165.
DEBUG: Inbound stream from 169.254.145.9:58165 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 120: t_isS_128K_8K
113/159 Test #120: t_isS_128K_8K ....................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:1155... done.
DEBUG: Outbound stream to 169.254.144.9:1155 finished (0 bytenew connection from s still in flight).
169.254.145.9:44059.
DEBUG: Inbound stream from 169.254.145.9:44059 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 121: t_isS_16_1
114/159 Test #121: t_isS_16_1 .......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:31740... done.
DEBUG: Outbound stream to 169.254.144.9:31740 finished (0 bytes still in flight).
new connection from 169.254.145.9:38190.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 122: t_isS_32K_d
115/159 Test #122: t_isS_32K_d ......................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:24496... done.
DEBUG: Outbound stream to 169.254.144.9:24496 finished (0 bytes still in flight).
new connection from 169.254.145.9:60799.
DEBUG: Inbound stream from 169.254.145.9:60799 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 123: t_isR_1M_32k
116/159 Test #123: t_isR_1M_32k .....................***Failed    0.24 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:2101... done.
new connection from 169.254.145.9:35513.
DEBUG: Outbound stream to 169.254.145.9:35513 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:2101 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3263 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 124: t_isR_128K_8K
117/159 Test #124: t_isR_128K_8K ....................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:14532... done.
new connection from 169.254.145.9:54686.
DEBUG: Outbound stream to 169.254.145.9:54686 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:14532 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3283 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 125: t_isR_16_1
118/159 Test #125: t_isR_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:25096... done.
new connection from 169.254.145.9:50543.
DEBUG: Outbound stream to 169.254.145.9:50543 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3303 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 126: t_isR_32K_d
119/159 Test #126: t_isR_32K_d ......................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:30840... done.
new connection from 169.254.145.9:42453.
DEBUG: Outbound stream to 169.254.145.9:42453 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:30840 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3324 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 127: t_isD_1M_32k
120/159 Test #127: t_isD_1M_32k .....................***Failed    0.29 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:23690... done.
new connection from 169.254.145.9:31294.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3345 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 128: t_isD_128K_8K
121/159 Test #128: t_isD_128K_8K ....................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:18359... done.
new connection from 169.254.145.9:19732.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3364 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3366 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 129: t_isD_16_1
122/159 Test #129: t_isD_16_1 .......................***Failed    0.17 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:9017... done.
new connection from 169.254.145.9:27539.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3387 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 130: t_isD_32K_d
123/159 Test #130: t_isD_32K_d ......................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:18019... done.
new connection from 169.254.145.9:26414.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3406 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect
/mnt/i/DIY/CS144/sponge/txrx.sh: line 156:  3404 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"

        Start 131: t_icS_128K_8K_l
124/159 Test #131: t_icS_128K_8K_l ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:9923... done.
new connection from 169.254.144.9:55075.
DEBUG: Outbound stream to 169.254.144.9:55075 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:9923 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3426 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 132: t_icS_128K_8K_L
125/159 Test #132: t_icS_128K_8K_L ..................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:4640... done.
new connection from 169.254.144.9:27472.
DEBUG: Outbound stream to 169.254.144.9:27472 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:4640 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3446 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 133: t_icS_128K_8K_lL
126/159 Test #133: t_icS_128K_8K_lL .................***Failed    0.18 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:2595... done.
new connection from 169.254.144.9:63738.
DEBUG: Outbound stream to 169.254.144.9:63738 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:2595 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3466 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 134: t_icR_128K_8K_l
127/159 Test #134: t_icR_128K_8K_l ..................***Timeout  10.16 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:19294... done.
DEBUG: Outbound stream to 169.254.145.9:19294 finished (0 bytes still in flight).
new connection from 169.254.144.9:14321.
DEBUG: Inbound stream from 169.254.144.9:14321 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 135: t_icR_128K_8K_L
128/159 Test #135: t_icR_128K_8K_L ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:7953... done.
DEBUG: Outbound stream to 169.254.145.9:7953 finished (0 bytes still in flight).
new connection from 169.254.144.9:8373.
DEBUG: Inbound stream from 169.254.144.9:8373 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 136: t_icR_128K_8K_lL
129/159 Test #136: t_icR_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:32469... done.
DEBUG: Outbound stream to 169.254.145.9:32469 finished (0 bytes still in flight).
new connection from 169.254.144.9:2986.
DEBUG: Inbound stream from 169.254.144.9:2986 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 137: t_icD_128K_8K_l
130/159 Test #137: t_icD_128K_8K_l ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:10145... done.
new connection from 169.254.144.9:13242.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3586 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 138: t_icD_128K_8K_L
131/159 Test #138: t_icD_128K_8K_L ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:24447... done.
new connection from 169.254.144.9:59234.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3605 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3607 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 139: t_icD_128K_8K_lL
132/159 Test #139: t_icD_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.145.9:19333... done.
new connection from 169.254.144.9:2820.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3626 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3628 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 140: t_isS_128K_8K_l
133/159 Test #140: t_isS_128K_8K_l ..................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:4627... 

        Start 141: t_isS_128K_8K_L
134/159 Test #141: t_isS_128K_8K_L ..................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:11195... done.
DEBUG: Outbound stream to 169.254.144.9:11195 finished (new connection from 169.254.145.9:14895.
0 bytes still in flight).
DEBUG: Inbound stream from 169.254.145.9:14895 finished cleanly.
DEBUG: Waiting for clean shutdown... Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 142: t_isS_128K_8K_lL
135/159 Test #142: t_isS_128K_8K_lL .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:4997... done.
DEBUG: Outbound stream to 169.254.144.9:4997 finished (0 bytes still in flight).
new connection from 169.254.145.9:51723.
DEBUG: Inbound stream from 169.254.145.9:51723 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 143: t_isR_128K_8K_l
136/159 Test #143: t_isR_128K_8K_l ..................***Failed    0.21 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:1430... done.
new connection from 169.254.145.9:54395.
DEBUG: Outbound stream to 169.254.145.9:54395 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:1430 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3745 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 144: t_isR_128K_8K_L
137/159 Test #144: t_isR_128K_8K_L ..................***Failed    0.22 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:18155... done.
new connection from 169.254.145.9:11114.
DEBUG: Outbound stream to 169.254.145.9:11114 finished (0 bytes still in flight).
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3768 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 145: t_isR_128K_8K_lL
138/159 Test #145: t_isR_128K_8K_lL .................***Failed    0.19 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:30207... done.
new connection from 169.254.145.9:4503.
DEBUG: Outbound stream to 169.254.145.9:4503 finished (0 bytes still in flight).
DEBUG: Inbound stream from 169.254.144.9:30207 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3788 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 146: t_isD_128K_8K_l
139/159 Test #146: t_isD_128K_8K_l ..................***Failed    0.23 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:20699... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3809 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 147: t_isD_128K_8K_L
140/159 Test #147: t_isD_128K_8K_L ..................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:33120... done.
new connection from 169.254.145.9:2753.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3827 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3829 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 148: t_isD_128K_8K_lL
141/159 Test #148: t_isD_128K_8K_lL .................***Failed    0.20 sec
DEBUG: Listening for incoming connection... DEBUG: Connecting to 169.254.144.9:24071... done.
new connection from 169.254.145.9:62116.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3848 Aborted                 coproc COPROC $3 -l $4 ${SERVER_PORT} > "$1" < "$2"
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3850 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 149: t_icnS_128K_8K_l
142/159 Test #149: t_icnS_128K_8K_l .................***Failed    0.26 sec
DEBUG: Connecting to 169.254.144.1:30995... done.
DEBUG: Inbound stream from 169.254.144.1:30995 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3870 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 150: t_icnS_128K_8K_L
143/159 Test #150: t_icnS_128K_8K_L .................***Failed    0.24 sec
DEBUG: Connecting to 169.254.144.1:27872... done.
DEBUG: Inbound stream from 169.254.144.1:27872 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  3889 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 151: t_icnS_128K_8K_lL
144/159 Test #151: t_icnS_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Connecting to 169.254.144.1:28983... done.

        Start 152: t_icnR_128K_8K_l
145/159 Test #152: t_icnR_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:13114... done.
DEBUG: Outbound stream to 169.254.144.1:13114 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:13114 has been fully acknowledged.

        Start 153: t_icnR_128K_8K_L
146/159 Test #153: t_icnR_128K_8K_L .................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:32273... done.
DEBUG: Outbound stream to 169.254.144.1:32273 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:32273 has been fully acknowledged.

        Start 154: t_icnR_128K_8K_lL
147/159 Test #154: t_icnR_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Connecting to 169.254.144.1:16879... done.
DEBUG: Outbound stream to 169.254.144.1:16879 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:16879 has been fully acknowledged.

        Start 155: t_icnD_128K_8K_l
148/159 Test #155: t_icnD_128K_8K_l .................***Failed    0.25 sec
DEBUG: Connecting to 169.254.144.1:2198... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  4043 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 156: t_icnD_128K_8K_L
149/159 Test #156: t_icnD_128K_8K_L .................***Failed    0.25 sec
DEBUG: Connecting to 169.254.144.1:17158... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  4063 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 157: t_icnD_128K_8K_lL
150/159 Test #157: t_icnD_128K_8K_lL ................***Failed    0.27 sec
DEBUG: Connecting to 169.254.144.1:31461... done.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception
/mnt/i/DIY/CS144/sponge/txrx.sh: line 110:  4083 Aborted                 $3 $4 ${SERVER_PORT} > "$1" < "$2"
Error in _rt_connect

        Start 158: t_isnS_128K_8K_l
151/159 Test #158: t_isnS_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:58800.
DEBUG: Inbound stream from 169.254.144.1:58800 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 159: t_isnS_128K_8K_L
152/159 Test #159: t_isnS_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:35456.
DEBUG: Inbound stream from 169.254.144.1:35456 finished cleanly.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 160: t_isnS_128K_8K_lL
153/159 Test #160: t_isnS_128K_8K_lL ................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... 

        Start 161: t_isnR_128K_8K_l
154/159 Test #161: t_isnR_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:57606.
DEBUG: Outbound stream to 169.254.144.1:57606 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:57606 has been fully acknowledged.

        Start 162: t_isnR_128K_8K_L
155/159 Test #162: t_isnR_128K_8K_L .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:59284.
DEBUG: Outbound stream to 169.254.144.1:59284 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:59284 has been fully acknowledged.

        Start 163: t_isnR_128K_8K_lL
156/159 Test #163: t_isnR_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:51902.
DEBUG: Outbound stream to 169.254.144.1:51902 finished (0 bytes still in flight).
DEBUG: Outbound stream to 169.254.144.1:51902 has been fully acknowledged.

        Start 164: t_isnD_128K_8K_l
157/159 Test #164: t_isnD_128K_8K_l .................***Timeout  10.01 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:36560.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 165: t_isnD_128K_8K_L
158/159 Test #165: t_isnD_128K_8K_L .................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:39240.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception

        Start 166: t_isnD_128K_8K_lL
159/159 Test #166: t_isnD_128K_8K_lL ................***Timeout  10.00 sec
DEBUG: Listening for incoming connection... new connection from 169.254.144.1:53650.
Exception in TCPConnection runner thread: TCPConnection::write() accepted less than advertised length
terminate called after throwing an instance of 'std::exception'
  what():  std::exception


22% tests passed, 124 tests failed out of 159

Total Test time (real) = 498.39 sec

The following tests FAILED:
	 33 - t_active_close (Failed)
	 34 - t_passive_close (Failed)
	 36 - t_ack_rst (Failed)
	 38 - t_ack_rst_win (Failed)
	 43 - t_winsize (Failed)
	 45 - t_retx (Failed)
	 46 - t_retx_win (Failed)
	 47 - t_loopback (Failed)
	 48 - t_loopback_win (Failed)
	 49 - t_reorder (Failed)
	 53 - t_udp_client_send (Failed)
	 54 - t_udp_server_send (Timeout)
	 55 - t_udp_client_recv (Timeout)
	 56 - t_udp_server_recv (Failed)
	 57 - t_udp_client_dupl (Failed)
	 58 - t_udp_server_dupl (Failed)
	 59 - t_ucS_1M_32k (Failed)
	 60 - t_ucS_128K_8K (Failed)
	 61 - t_ucS_16_1 (Failed)
	 62 - t_ucS_32K_d (Failed)
	 63 - t_ucR_1M_32k (Timeout)
	 64 - t_ucR_128K_8K (Timeout)
	 65 - t_ucR_16_1 (Timeout)
	 66 - t_ucR_32K_d (Timeout)
	 67 - t_ucD_1M_32k (Failed)
	 68 - t_ucD_128K_8K (Failed)
	 69 - t_ucD_16_1 (Failed)
	 70 - t_ucD_32K_d (Failed)
	 71 - t_usS_1M_32k (Timeout)
	 72 - t_usS_128K_8K (Timeout)
	 73 - t_usS_16_1 (Timeout)
	 74 - t_usS_32K_d (Timeout)
	 75 - t_usR_1M_32k (Failed)
	 76 - t_usR_128K_8K (Failed)
	 77 - t_usR_16_1 (Failed)
	 78 - t_usR_32K_d (Failed)
	 79 - t_usD_1M_32k (Failed)
	 80 - t_usD_128K_8K (Failed)
	 81 - t_usD_16_1 (Failed)
	 82 - t_usD_32K_d (Failed)
	 83 - t_ucS_128K_8K_l (Failed)
	 84 - t_ucS_128K_8K_L (Failed)
	 85 - t_ucS_128K_8K_lL (Failed)
	 86 - t_ucR_128K_8K_l (Timeout)
	 87 - t_ucR_128K_8K_L (Timeout)
	 88 - t_ucR_128K_8K_lL (Timeout)
	 89 - t_ucD_128K_8K_l (Failed)
	 90 - t_ucD_128K_8K_L (Failed)
	 91 - t_ucD_128K_8K_lL (Failed)
	 92 - t_usS_128K_8K_l (Timeout)
	 93 - t_usS_128K_8K_L (Timeout)
	 94 - t_usS_128K_8K_lL (Timeout)
	 95 - t_usR_128K_8K_l (Failed)
	 96 - t_usR_128K_8K_L (Failed)
	 97 - t_usR_128K_8K_lL (Failed)
	 98 - t_usD_128K_8K_l (Failed)
	 99 - t_usD_128K_8K_L (Failed)
	100 - t_usD_128K_8K_lL (Failed)
	101 - t_ipv4_client_send (Failed)
	102 - t_ipv4_server_send (Timeout)
	103 - t_ipv4_client_recv (Timeout)
	104 - t_ipv4_server_recv (Failed)
	105 - t_ipv4_client_dupl (Failed)
	106 - t_ipv4_server_dupl (Failed)
	107 - t_icS_1M_32k (Failed)
	108 - t_icS_128K_8K (Failed)
	109 - t_icS_16_1 (Failed)
	110 - t_icS_32K_d (Failed)
	111 - t_icR_1M_32k (Timeout)
	112 - t_icR_128K_8K (Timeout)
	113 - t_icR_16_1 (Timeout)
	114 - t_icR_32K_d (Timeout)
	115 - t_icD_1M_32k (Failed)
	116 - t_icD_128K_8K (Failed)
	117 - t_icD_16_1 (Failed)
	118 - t_icD_32K_d (Failed)
	119 - t_isS_1M_32k (Timeout)
	120 - t_isS_128K_8K (Timeout)
	121 - t_isS_16_1 (Timeout)
	122 - t_isS_32K_d (Timeout)
	123 - t_isR_1M_32k (Failed)
	124 - t_isR_128K_8K (Failed)
	125 - t_isR_16_1 (Failed)
	126 - t_isR_32K_d (Failed)
	127 - t_isD_1M_32k (Failed)
	128 - t_isD_128K_8K (Failed)
	129 - t_isD_16_1 (Failed)
	130 - t_isD_32K_d (Failed)
	131 - t_icS_128K_8K_l (Failed)
	132 - t_icS_128K_8K_L (Failed)
	133 - t_icS_128K_8K_lL (Failed)
	134 - t_icR_128K_8K_l (Timeout)
	135 - t_icR_128K_8K_L (Timeout)
	136 - t_icR_128K_8K_lL (Timeout)
	137 - t_icD_128K_8K_l (Failed)
	138 - t_icD_128K_8K_L (Failed)
	139 - t_icD_128K_8K_lL (Failed)
	140 - t_isS_128K_8K_l (Timeout)
	141 - t_isS_128K_8K_L (Timeout)
	142 - t_isS_128K_8K_lL (Timeout)
	143 - t_isR_128K_8K_l (Failed)
	144 - t_isR_128K_8K_L (Failed)
	145 - t_isR_128K_8K_lL (Failed)
	146 - t_isD_128K_8K_l (Failed)
	147 - t_isD_128K_8K_L (Failed)
	148 - t_isD_128K_8K_lL (Failed)
	149 - t_icnS_128K_8K_l (Failed)
	150 - t_icnS_128K_8K_L (Failed)
	151 - t_icnS_128K_8K_lL (Timeout)
	152 - t_icnR_128K_8K_l (Timeout)
	153 - t_icnR_128K_8K_L (Timeout)
	154 - t_icnR_128K_8K_lL (Timeout)
	155 - t_icnD_128K_8K_l (Failed)
	156 - t_icnD_128K_8K_L (Failed)
	157 - t_icnD_128K_8K_lL (Failed)
	158 - t_isnS_128K_8K_l (Timeout)
	159 - t_isnS_128K_8K_L (Timeout)
	160 - t_isnS_128K_8K_lL (Timeout)
	161 - t_isnR_128K_8K_l (Timeout)
	162 - t_isnR_128K_8K_L (Timeout)
	163 - t_isnR_128K_8K_lL (Timeout)
	164 - t_isnD_128K_8K_l (Timeout)
	165 - t_isnD_128K_8K_L (Timeout)
	166 - t_isnD_128K_8K_lL (Timeout)
Errors while running CTest
CMakeFiles/check_lab4.dir/build.make:57: recipe for target 'CMakeFiles/check_lab4' failed
make[3]: *** [CMakeFiles/check_lab4] Error 8
CMakeFiles/Makefile2:131: recipe for target 'CMakeFiles/check_lab4.dir/all' failed
make[2]: *** [CMakeFiles/check_lab4.dir/all] Error 2
CMakeFiles/Makefile2:138: recipe for target 'CMakeFiles/check_lab4.dir/rule' failed
make[1]: *** [CMakeFiles/check_lab4.dir/rule] Error 2
Makefile:155: recipe for target 'check_lab4' failed
make: *** [check_lab4] Error 2
